app/src/views/GrantRound.vue

```diff
@@ -75,25 +75,37 @@
     <!-- Interactions Bar for Share, Add Funds  -->
     <div class="px-4 md:px-12 py-8 border-b border-grey-100">
       <div class="flex flex-wrap gap-x-6 gap-y-4">
-        <!-- TODO: Share-->
-        <div class="flex items-center gap-x-2 cursor-pointer group ml-auto">
-          <ShareIcon class="icon icon-primary icon-small" />
-          <span class="text-grey-400 group-hover:text-grey-500">Share</span>
-        </div>
+        <!-- Tweet -->
+        <a
+          target="_blank"
+          :href="
+            'https://twitter.com/intent/tweet?text=' +
+            grantRoundMetadata?.name +
+            '&url=https://grants.gtcdao.net' +
+            $route.path
+          "
+          class="flex items-center gap-x-2 cursor-pointer group ml-auto"
+        >
+          <TweetIcon class="icon icon-primary icon-small" />
+          <span class="text-grey-400 group-hover:text-grey-500">Tweet</span>
+        </a>
         <!-- Add funds to Grant Round -->
         <div @click="showAddFunds()" class="flex items-center gap-x-2 cursor-pointer group">
-          <DonateIcon class="icon icon-primary icon-small icon-secondary" />
+          <DonateIcon class="icon icon-primary icon-small" />
           <span class="text-grey-400 group-hover:text-grey-500">Add funds</span>
         </div>
       </div>
     </div>
 
     <!-- Description -->
     <SectionHeader title="Description" />
+
     <div class="border-b border-grey-100">
-      <p style="white-space: pre-line" class="intent px-4 md:px-12 py-24 mx-auto max-w-6xl">
-        {{ grantRoundMetadata?.description }}
-      </p>
+      <div class="px-4 md:px-12 py-12 mx-auto max-w-6xl">
```

Comment:
This does mean everything would be not aligned right. 
I'd say keep white-space: preline until you've thrown in a CRLF / ENTER 

Right now line breaks as stored as `/n` and we don't want to store the entire HTML of description onto IPFS

Reply:
changed to "whitespace-pre-line"  but later maybe revisit to have it as i want in design.